modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/pod-logs.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if ingressNginx }}
```

Comment:
Not sure in this,
With the for loop, if `extra_namespaces` contains `ingress-nginx` the bool will eq `true`
hence the config will apply? Will need to set it so it will use the pod-logs.yaml and monitors.yaml when it eq `false`.